test/tbb/test_concurrent_monitor.cpp

```diff
@@ -84,7 +84,14 @@ TEST_CASE("Stress test") {
                 }
                 case 1:
                 {
+#if (__TBB_GCC_VERSION >= 120100 && __TBB_GCC_VERSION < 140000 ) && !__clang__ && !__INTEL_COMPILER
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wstringop-overflow"
```

Comment:
Could you please add a comment on what actual warning are we going to suppress? 
It is unclear how `notify_one` and `-Wstringop-overflow` are related.
It would be easier in the future to detect workarounds in our code if similar warnings will appear in the later gcc versions.

Reply:
Added.